### ISSUE TITLE
Edge Detection using Convolutions from scratch

### ISSUE THREAD

**Issue by nishant-nayak** [WebClub-NITK/Hacktoberfest-2k20#43]
### Description
Detect edges in an image of your choice by manipulating pixel values/applying convolutions, and upload the original image as well as the image with the detected edges, along with the code used.

### Details
- Technical Specifications: Preferred language to use is Python, and some helpful libraries are NumPy and OpenCV.
- Type of issue: Single
- Time Limit: 1 day after assignment of the issue

### Issue requirements / progress
1. [ ] Write the code to detect edges from scratch
2. [ ] Upload the code, original image and image with edges detected

### Resources
List of resources that might be required / helpful.
- [Kernels in Image Processing](https://en.wikipedia.org/wiki/Kernel_(image_processing))
- [NumPy Documentation](https://numpy.org/doc/stable/)

### Directory Structure
Create a folder named `43_edge_detection` in the "Others" folder (if it doesn't already exist) and place the appropriately named files inside this folder.

#### Note
1. Please claim the issue first by commenting here before starting to work on it.
2. Once you are done with the task and have created a Pull Request, please tag @nishant-nayak to request a review.

**Comment by DineshBS44** [comment]
@nishant-nayak, I would like to work on this issue. I'm planning on using Canny Edge filter for detection of edges in the image. I'll code it in python.

**Comment by nishant-nayak** [comment]
@DineshBS44 sure! Go ahead, and feel free to ask if you have any doubts/queries.

**Comment by DineshBS44** [comment]
@nishant-nayak, I'm done with this task. But, there's another task which was assigned to me earlier in the same repo and I made a PR for that task a few hours ago. Now, I'm not able to make a PR for this task yet because the reviewer hasn't merged the previous PR yet. So, I have no choice but to wait until that PR is merged. I'm saying this because the deadline for this task is 1 day.

**Comment by krithikvaidya** [comment]
> @nishant-nayak, I'm done with this task. But, there's another task which was assigned to me earlier in the same repo and I made a PR for that task a few hours ago. Now, I'm not able to make a PR for this task yet because the reviewer hasn't merged the previous PR yet. So, I have no choice but to wait until that PR is merged. I'm saying this because the deadline for this task is 1 day.

Hey, your previous PR was merged, you can go ahead with this now

**Comment by DineshBS44** [comment]
@nishant-nayak , I opened a Pull Request, please check.

**Comment by nishant-nayak** [comment]
@DineshBS44 thanks for contributing! We'll merge your PR ASAP.

**Comment by krithikvaidya** [comment]
@nishant-nayak I'll close the issue since the PR has been merged.
